ucsb_ds_capstone_projects_2021/projects/nceas/update2.md

```diff
@@ -21,33 +21,58 @@ February 28, 2021
     - Sam: [Bonanza](https://github.com/TokyoExpress/bonanza)
 
 
-###### Examples of Visualizations in our vignettes, all of which were produced using ggplot from the tidyverse package:
+###### Examples of our vignettes:
 
 **Sophia** \
-This image shows an initial view of the Bull Shark Count Overtime. This data is from the Florida Coastal Everglades (FCE) LTER Site. I thought this was an interesting plot to include because of the pattern happening YoY. 
-![shark](images/update2sophia1.png)
-
-The second image shows the relationship between Temperature and Bull Sharks. I thought this was important to include because it brings up an important point about the nature of collecting data and taking data with a grain of salt. From a first glance, it appears that more Bull Sharks are counted when the Temperature is higher; however researchers are also more likely to survey when the Temperature is warmer/nicer and that may be a bias in the data and affect the number of Bull Sharks counted.
-![shark2](images/update2sophia2.png)
+I worked with a Fish Abundance dataset from the SBC (Santa Barbara Coastal) LTER. This dataset is particularly interesting because researchers record the date, site, fish type, and length. I chose this visualization of garibaldi fish due to the variety of its recorded length across multiple sites within SBC. The disparity in garibaldi fish length could be due to actual differences or also due to inconsistency in researcher's measuring techniques. This is a great real-world example for future data science students to learn from due to these clear, inherent differences. The fish abundance dataset is extremely workable and had a lot of promise for future analysis.
+![fish](images/update2sophia3.png) 
 
 **Lia** \
-I choose a dataset from the Moorea Coral Reef site that analyzes the changes in abundance of fish functional groups in relation to live/dead coral. My initial visualization was to showcase the obvious: how live coral rates decrease over time. Then, I plotted this per site to maybe account for biases in the data that were unobserved otherwise. We concluded that coral rates continue to decrease (yet as a slower rate), despite what the initial visualization might imply. 
-![coral](images/update2lia1.png)
-![coral2](images/update2lia2.png)
+The dataset I worked with is from the Luquillo site. Luquillo is an experimental forest. After a hurricane Georges in 1998, the dataset I explored investigates the recovery of snails over time. After converted the data from wide to long, I aggregated total snail counts by species and plotted them in a time series to try and see what snail species seemed the most resilient. 
+
+![image](https://user-images.githubusercontent.com/55970232/111717307-071d8d00-8815-11eb-9260-8141544e403e.png)
```

Comment:
Place the images in `images/` directory and embed the images using the markdown notation as was done previously.